ddr_artifacts.mk

```diff
@@ -30,10 +30,10 @@ top_srcdir = .
 include $(top_srcdir)/omrmakefiles/configure.mk
 
 # Ensure the source tree path is canonically-formed and ends with /
-abs_srcroot = $(abspath $(TOP_SRCDIR))/
+abs_srcroot := $(subst //,/,$(abspath $(TOP_SRCDIR))/)
 
 all: check-vars
-	./tools/ddrgen/src/macros/getMacros.sh $(abs_srcroot)
+	bash $(top_srcdir)/ddr/tools/getmacros $(abs_srcroot)
```

Comment:
should we be relying on `bash`?  Can we not just use `sh`?  I know there are sometimes issues with `bash` on AIX and zOS.

Reply:
getmacros begins with
```sh
#!/bin/bash
```
and I think it does depend on features in `bash` that are not in `sh`.

Reply:
makes sense then to use `bash`